unlock-app/src/pages/log.js

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import UnlockPropTypes from '../propTypes'
+import Layout from '../components/interface/Layout'
+import CreatorAccount from '../components/creator/CreatorAccount'
+import BrowserOnly from '../components/helpers/BrowserOnly'
+import GlobalErrorConsumer from '../components/interface/GlobalErrorConsumer'
+import GlobalErrorProvider from '../utils/GlobalErrorProvider'
+import { pageTitle } from '../constants'
+import configure from '../config'
+
+const { chainExplorerUrlBuilders } = configure()
```

Comment:
It's better if you use the ConfigContext to get config values

Reply:
In this case I need the value from the config in `mapStateToProps`. I'm not super familiar with HOCs; what's the best way to connect those?

Reply:
use `withConfig` around `connect` and it will pass it in as a prop named `config`. Then you don't need to wrangle with the context either (and I had forgotten about that too because I'm old)